plugins/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -29,17 +29,23 @@
             - Name of the retention policy.
         required: true
         type: str
+    state:
+        description:
+            - State of the retention policy.
+        choices: [ absent, present ]
+        default: present
+        type: str
     duration:
         description:
             - Determines how long InfluxDB should keep the data. If specified, it
               should be C(INF) or at least one hour. If not specified, C(INF) is
               assumed. Supports complex duration expressions with multiple units.
-        required: true
+            - Required only if C(state) is set to C(present).
```

Comment:
```suggestion
            - Required only if I(state) is set to C(present).
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation